style.css

```diff
@@ -112,13 +123,44 @@ header {
   line-height: 52px;
 }
 
-.placeholder {
+.placeholder-multistories {
+  display: flex;
+  justify-content: left;
+  margin-top: 80px;
+  margin-left: 1.5rem;
   margin-right: 1.5rem;
 }
 
-.sepration-line {
-  margin-top: 30px;
-  margin-bottom: 80px;
+.placeholder {
+  max-width: 100%;
+  max-height: 100%;
+}
+
+.recent-work {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: repeat(1, 1fr);
+}
+
+.multistories {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  margin: 1.5rem;
+}
+
+.multistories > h2 {
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+}
+
```

Comment:
- [OPTIONAL] There's no consistency in the margins(right and left) maintained throughout all the projects under recent works sections. The width used for the first project is wider as opposed to the rest of the projects. And this shows a great difference in your design compare to the Figma design. Kindly take a look, compare and work your design alongside the Figma design.

### Your Design
![your-projects-consistency](https://user-images.githubusercontent.com/26597432/220605103-a2366bd6-cfa0-43cc-ba67-3753cc608a68.png)

### Figma Design
![temp-projects-spacing](https://user-images.githubusercontent.com/26597432/220605408-15dff722-33dc-448d-ad69-0784465823f2.png)